microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -81,7 +81,7 @@ def command(self) -> str:
 
     _redhat_packages = [
         "psmisc",
-        "kernel-devel-$(uname -r)",
+        "kernel-devel",
```

Comment:
This might install kernel devel packages for a different kernel version. Have you checked if this scenario happens?

Reply:
For the cases where there is no `kernel-devel-$(uname -r)` package, it does install a package for a slightly different kernel build. For example, `4.18.0-240.10.1.el8_3.x86_64` installs `kernel-devel-4.18.0-348.7.1.el8_5.x86_64`.

I have not checked whether there are any cases where `kernel-devel-$(uname -r)` exists but `kernel-devel` does not install that package. From my experience with the InfiniBand test cases, it is better to trust the package manager to select the best kernel-devel package to install. The only case where this is not true is if the kernel has been updated but not rebooted yet so the kernel version might not match.

Reply:
If there are any problems with this, then we would need to try installing `kernel-devel-$(uname -r)` first then if that fails, then try installing `kernel-devel`.

Reply:
Agreed. Retrying without `uname -r` on failure would be more robust.